app/controllers/submit.js

```diff
@@ -733,6 +749,7 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                         }
                         model.save()
                             .then(() => {
+                                this.set('node.description', this.get('node.description'));
```

Comment:
See above comment- setting a value we already have seems unnecessary, and may have side effects.